css/main.css

```diff
@@ -356,4 +356,19 @@ li {
   list-style: inside url("../media/svg/circle.svg");
 }
 
+footer {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding: 10%;
+}
 
+.button {
```

Comment:
Kindly update your button element to include button interaction styling when `hovered ` and `pressed`🛠️. You can use `.button:hover` and `.button:active` to style such interactions. Please ensure to match the design guide provided by figma 🎨

Figma design:
<img width="367" alt="image" src="https://user-images.githubusercontent.com/37116322/218156677-943c0695-d427-4b91-816d-76711efd13c2.png">